packaging/check-kernel-config.sh

```diff
@@ -51,15 +51,18 @@ fi
 
 if [ -n "${CONFIG_PATH}" ]; then
   GREP='grep'
+  CAT='cat'
 
   if echo "${CONFIG_PATH}" | grep -q '.gz'; then
-    GREP='zgrep'
+    CAT='zcat'
   fi
 
   REQUIRED_CONFIG="KPROBES KPROBES_ON_FTRACE HAVE_KPROBES BPF BPF_SYSCALL BPF_JIT"
 
   for required_config in ${REQUIRED_CONFIG}; do
-    if ! "${GREP}" -q "CONFIG_${required_config}=y" "${CONFIG_PATH}"; then
+    # Fix issue https://github.com/netdata/netdata/issues/14668
+    # if ! "${GREP}" -q "CONFIG_${required_config}=y" "${CONFIG_PATH}"; then
+    if "${CAT}" "${CONFIG_PATH}" | "${GREP}" -q "CONFIG_${required_config}=y" >&2 /dev/null ;then
```

Comment:
```suggestion
    if ! { "${CAT}" "${CONFIG_PATH}" | "${GREP}" -q "CONFIG_${required_config}=y" >&2 >/dev/null } ;then
```

Reply:
Hello @Ferroin ,

Your suggestion raises the following warning:

```sh
  if ! { "${CAT}" "${CONFIG_PATH}" | "${GREP}" -q "CONFIG_${required_config}=y" >&2 >/dev/null } ;then
    ^-- SC1009 (info): The mentioned syntax error was in this if expression.
         ^-- SC1073 (error): Couldn't parse this brace group. Fix to allow more checks.
                                                                                                 ^-- SC1083 (warning): This } is literal. Check expression (missing ;/\n?) or quote it.
                                                                                                    ^-- SC1056 (error): Expected a '}'. If you have one, try a ; or \n in front of it.
                                                                                                        ^-- SC1072 (error): Unexpected keyword/token. Fix any mentioned problems and try again.
```

Reply:
Quite so, forgot the `;` that should be after the `/dev/null` but before the `}`.

Reply:
Irrespective of how you fix the condition though, the condition is backwards (you’re running the body of the `if` clause when the option is present, not when it’s absent), and you’re not actually redirecting to `/dev/null`.

Reply:
Thank you, last commit brought the fix.